runelite-client/src/main/java/net/runelite/client/plugins/itemstats/ItemStatChanges.java

```diff
@@ -90,7 +90,9 @@ private void init()
 		add(food(perc(.1, 1)), COOKED_SWEETCORN, SWEETCORN_7088 /* Bowl of cooked sweetcorn */);
 		add(combo(food(1), boost(DEFENCE, perc(.02, 1))), CABBAGE_1967 /* Draynor Manor */);
 		add(combo(2, food(4), heal(RUN_ENERGY, 5)), PAPAYA_FRUIT);
-
+		add(range(food(5), food(7)), THIN_SNAIL_MEAT);
+		add(range(food(7), food(9)), FAT_SNAIL_MEAT);
+		
```

Comment:
Remove the trailing space on this line.

Reply:
This seems like a git tracking issue, there was no line added or removed and it's following the same pattern as the entire file. So I would think we would want to leave it this way.

Reply:
Its because the previous newline was empty, yours has tabs on it

Reply:
`		` he means there is this on this empty line, so it's not really empty